src/AutoTable.jsx

```diff
@@ -1,14 +1,38 @@
 import React from "react";
 import PropTypes from "prop-types";
-import withData from "withdata";
-import { isEqual } from "lodash";
+import { Data } from "perch-data";
+import { TableRow, TableCell, Typography } from "material-ui";
 import { BaseTable as Table, LoadingRow } from "./";
 
+const ErrorRow = ({ columnCount }) => (
+  <TableRow>
+    <TableCell colSpan={columnCount}>
+      <Typography type="caption" align="center">
+        An error occurred while fetching this data.
+      </Typography>
+    </TableCell>
+  </TableRow>
+);
+
+ErrorRow.propTypes = {
+  columnCount: PropTypes.number.isRequired
+};
+
+const getPaginationForData = data =>
+  data && data.total_count > 0
+    ? {
+        count: data.total_count,
+        rowsPerPage: data.page_size,
+        page: data.page_number,
+        onChangePage: this.handleChangePage
```

Comment:
This gets compiled to `undefined.handleChangePage`

Reply:
Fixed! Ty